bin/node/runtime/src/lib.rs

```diff
@@ -898,10 +918,8 @@ impl pallet_tips::Config for Runtime {
 }
 
 parameter_types! {
-	pub ContractDeposit: Balance = deposit(
-		1,
-		<pallet_contracts::Pallet<Runtime>>::contract_info_size(),
-	);
+	pub const DepositPerItem: Balance = deposit(1, 0);
+	pub const DepositPerByte: Balance = deposit(0, 1);
```

Comment:
["Calling a contract will transfer balance from the caller to the contract or vise versa depending on whether the call increased or decreased the storage usage"
](https://github.com/paritytech/substrate/issues/9807)